FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -458,14 +490,21 @@ - (BOOL)handleUniversalLink:(NSURL *)universalLinkURL
                 }];
     return YES;
   } else {
-    FIRDynamicLink *dynamicLink = [self dynamicLinkFromUniversalLinkURL:universalLinkURL];
-    if (dynamicLink) {
-      completion(dynamicLink, nil);
-      return YES;
+    if ([self respondsToSelector:@selector(dynamicLinkFromUniversalLinkURL:completion:)]) {
```

Comment:
could you please explain why you think we need to check for the existence of "dynamicLinkFromUniversalLinkURL:completion" here?

Reply:
Well, initially I introduced it for compatibility purposes, but now I see that it's not needed.

Reply:
Thanks for removing.